src/main/resources/processes/IEDET/IEDET_PSU_TRIAGE.bpmn

```diff
@@ -29,20 +29,25 @@
       <bpmn:incoming>Flow_0rvtbmq</bpmn:incoming>
       <bpmn:outgoing>Flow_02ks0ls</bpmn:outgoing>
     </bpmn:serviceTask>
-    <bpmn:sequenceFlow id="Flow_02ks0ls" sourceRef="Activity_0dgy2k0" targetRef="Activity_1u4jmf5" />
+    <bpmn:sequenceFlow id="Flow_02ks0ls" sourceRef="Activity_0dgy2k0" targetRef="Service_UpdateDeadline" />
     <bpmn:task id="Activity_1u4jmf5" name="Return Case">
       <bpmn:extensionElements>
         <camunda:inputOutput>
           <camunda:outputParameter name="ReturnCase">${true}</camunda:outputParameter>
         </camunda:inputOutput>
       </bpmn:extensionElements>
-      <bpmn:incoming>Flow_02ks0ls</bpmn:incoming>
+      <bpmn:incoming>Flow_18kgsnq</bpmn:incoming>
       <bpmn:outgoing>Flow_09pdaeg</bpmn:outgoing>
     </bpmn:task>
     <bpmn:sequenceFlow id="Flow_09pdaeg" sourceRef="Activity_1u4jmf5" targetRef="EndEvent_Triage" />
     <bpmn:sequenceFlow id="Flow_1gidqty" name="" sourceRef="Gateway_0khr7oc" targetRef="EndEvent_Triage">
       <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${PsuTriageOutcome == "CloseCase"}</bpmn:conditionExpression>
     </bpmn:sequenceFlow>
+    <bpmn:serviceTask id="Service_UpdateDeadline" name="Update Case Deadline" camunda:expression="${bpmnService.updateDeadlineDays(execution.getVariable(&#34;CaseUUID&#34;),execution.processBusinessKey,20)}">
```

Comment:
I think it might be a bit better if this action was in PSU_IEDET_COMPLAINT instead then it wouldn't have to be duplicated for triage and outcome

Reply:
Incorporated the changes